### ISSUE TITLE
Rename a column

### ISSUE THREAD

**Issue by dperlman** [njvack/scorify#3]
There's not an obvious way to simply rename a column in the input file. You can hack it by doing something like measure newVar join(originalVar) (off the top of my head) but it took me a while to figure that out.

**Comment by njvack** [comment]
Yeah, the join() hack is clever but kind of bullshit. My sense is that the behavior should be something like:

* You get a `rename` directive; you'd use `rename, old_column, new_column`
* You can't rename something to an already-used column, that would be insane
* Renames are processed before any scoring or measures take place
* Once you've renamed a column, you must use the new name everywhere in the scoresheet

Thoughts?

**Comment by dperlman** [comment]
I like all that, but it raises one question I have wondered about when I was working on stimtime. Does it work better for the user if the directives are carried out in the order they are written, or if they have an internally determined order like what you describe here?

In this case the application domain is simple enough that it probably doesn't make much difference and the way you described is best. But like imagemagick is really useful for having in order processing. And it felt like stimtime was somewhere in between. 

Basically yes I agree with your plan.

**Comment by njvack** [comment]
Yeah, I think this is simple enough that it can be order agnostic. The main argument I can see for enforcing order is to keep people from doing something like:

```
score, foo_1, foo
...
rename, bar_1, foo_1
```

which is terrible. But I think the badness of it is self-evident enough.

**Comment by njvack** [comment]
OK, this is done in baf631fda977ad4e33714cba757dd8a13332c58a.

I haven't deployed it to our local servers yet, give it a try locally and see if it works for you first. 99% sure it will.
